src/components/NamespacedPage/NamespacedPage.tsx

```diff
@@ -40,10 +41,12 @@ const NamespacedPage: React.FunctionComponent<NamespacedPageProps> = ({ children
   }
 
   return (
-    <NamespaceContext.Provider value={{ namespace: activeNamepace }}>
-      <AppBanner />
-      <ErrorBoundary>{children}</ErrorBoundary>
-    </NamespaceContext.Provider>
+    <ModalProvider>
+      <NamespaceContext.Provider value={{ namespace: activeNamepace }}>
+        <AppBanner />
+        <ErrorBoundary>{children}</ErrorBoundary>
+      </NamespaceContext.Provider>
+    </ModalProvider>
```

Comment:
I think modal provider should be inside the namespace provider, so components in modal could have access to the namespace context.